src/ts/cmd/endpoint-registry/ui/i40-aas-registry-ui/webapp/mockserver/mockdata/SemanticProtocols/SemanticProtocols.json

```diff
@@ -0,0 +1,37 @@
+[{
```

Comment:
Depending on functionality consider moving to a test folder

Reply:
[Unused Json deleted](https://github.com/SAP/i40-aas/pull/182/commits/654733785652a1ba7a5aeeedc48c4925385f8ec3)